### PR TITLE
[Rebranding] Use Profile to describe brave profiles instead of People in MacOS

### PR DESCRIPTION
**Pull request by jagadeshjai**
- Use Profile to describe brave profiles instead of People in MacOS

Resolves https://github.com/brave/brave-browser/issues/37732

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by jagadeshjai** [comment]
@mkarolin / @sangwoo108 

Please take a look.

**Review by mkarolin — COMMENTED** [review]
This would have the opposite effect. The replacement here replaces the use of 'People' in Chromium strings with the word 'Profiles'. I believe the issue here is that if English is used as the UI language in India, it probably defaults to en-GB localization where the translation wasn't done correctly and 'People' was used instead of Profiles. I adjusted the translation on our localization site, but will need to verify locally.

**Comment by jagadeshjai** [comment]
> This would have the opposite effect. The replacement here replaces the use of 'People' in Chromium strings with the word 'Profiles'.

I initially thought that in the early stages of Brave, 'People' was used to refer to profiles. So that's why you included it in this replacement entry in the rebrand script.

>  I adjusted the translation on our localization site, but will need to verify locally.

I'm not entirely sure about the localization site. Is Brave using an API from this translation site to handle translations?
If yes, When will it get updated in .xtb files?

**Comment by mkarolin** [comment]
@jagadeshjai if you don't mind, please try pulling this commit into your branch https://github.com/brave/brave-core/commit/873d27b0a35b98d3430f45275bdc3b66ad8e3447 to verify if this fixes it (I don't have access to MacOS at the moment to check).

**Comment by jagadeshjai** [comment]
> @jagadeshjai if you don't mind, please try pulling this commit into your branch [873d27b](https://github.com/brave/brave-core/commit/873d27b0a35b98d3430f45275bdc3b66ad8e3447) to verify if this fixes it (I don't have access to MacOS at the moment to check).

Unfortunately, I don't have macOS at the moment 😔. I'll let you know if I can do something about this.

**Comment by mkarolin** [comment]
@jagadeshjai I did verify that the localisation change does fix the problem. Feel free to grab that commit to update your PR.

**Comment by jagadeshjai** [comment]
> @jagadeshjai I did verify that the localisation change does fix the problem. Feel free to grab that commit to update your PR.

Will do it @mkarolin , thanks for the patch!

File: script/lib/l10n/grd_string_replacements.py [hunk 1]
```diff
@@ -33,8 +33,6 @@
     (r'Incognito', r'Private'),
     (r'inco&gnito', r'&private'),
     (r'Inco&gnito', r'&Private'),
-    (r'People', r'Profiles'),
-    # 'people' but only in the context of profiles, not humans.
```
Comment (mkarolin) [inline script/lib/l10n/grd_string_replacements.py:37]: You don't need to remove this.

Comment (jagadeshjai) [reply script/lib/l10n/grd_string_replacements.py]: Done. Removed it.

**Review by mkarolin — APPROVED** [review]
LGTM

**Pull request merged**